cluster/addons/addon-manager/kube-addons.sh

```diff
@@ -159,6 +164,15 @@ function update_addons() {
   fi
 }
 
+function ensure_addons() {
+  # Create objects already exist should fail.
+  # Filter out `AlreadyExists` message to not noisily log.
+  ${KUBECTL} ${KUBECTL_OPTS} create --namespace=${SYSTEM_NAMESPACE} -f ${ADDON_PATH} -l ${SERVICE_ENSURE_EXIST_LABEL} \
```

Comment:
For the `kubectl` command, try to create something that already exist will exit with 255. And successfully create all objects should exit with 0. Though another common case would be some creations succeed while some creations fail. Not sure what the exit code would be in this case.

I removed the line "completed successfully" and "completed with errors" for this function because it is somehow hard to tell whether it run correctly or not.